### DISCUSSION TITLE
Get the checked column in 'Configure Columns'

### DISCUSSION THREAD

**Discussion by winex01** [orchidsoftware/platform#2766] [Q&A] [answered]
Is it possible to get whatever columns is check or shown in the Configure Columns?

![Screenshot 2023-12-14 081645](https://github.com/orchidsoftware/platform/assets/23657442/1c51b3bb-659e-41b5-9445-d8c745e8718e)

```php
// export method called from button

public function export()
{
     // get the shown/checked columns only
}
```

**Comment by macit-emre** [comment, marked as answer]
they are stored in browser localStorage. 
You may see it here : 
![Screenshot 2023-12-21 at 14 35 34](https://github.com/orchidsoftware/platform/assets/70325326/6b1802ad-e0d1-421c-9eae-e0665a78b54d)

So you may "technically" get from localStorage.

But as i understand, you want to use the data inside the "Screen" to avoid loading some expensive data. So you may not.

**Reply by winex01** [reply]
Hello there! My apologies for the delayed response; I've been away on a Christmas holiday and just returned.
